contrib/mypy/tests/python/pants_test/contrib/mypy/tasks/BUILD

```diff
@@ -8,5 +8,15 @@ python_tests(
     'tests/python/pants_test:int-test',
     'tests/python/pants_test:interpreter_selection_utils',
   ],
-  tags={'integration'},
-)
\ No newline at end of file
+  tags={'integration', 'integration_test'},
+)
+
+python_tests(
+  name='mypy',
+  source = 'test_mypy.py',
+  dependencies=[
+    'tests/python/pants_test:task_test_base',
+    'contrib/mypy/src/python/pants/contrib/mypy/tasks',
+  ],
+  tags = {'integration_test'},
```

Comment:
@wisechengyi do you know why this change was introduced?

Reply:
Do you mean the tag? It was probably because it started out as integration tests, but we kept unit in the end, so please feel free to remove.